extract_pov_edits/dataflow_main.py

```diff
@@ -156,18 +182,23 @@ def process(self, element, bucket):
     return
 
 
-class WriteDecompressedFile(beam.DoFn):
+class IngestDumps(beam.DoFn):
   def __init__(self):
       self.processed_revisions = Metrics.counter(self.__class__, 'processed_revisions')
-      self.errors = Metrics.counter(self.__class__, 'errors')
 
-
-  def process(self, element, bucket, ingestFrom):
-    """Ingests the xml dump into json, returns the josn records
+  def process(self, element, bucket, ingestFrom, all_edits):
+    """Ingests the xml dump into consecutive revision pairs in json format.
     """
-    nltk.download('punkt')
-    # Decompress the data dump
-    chunk_name = element
+    if isinstance(element, basestring):
+      chunk_name = element
+      page_ids = None
+    else:
+      # If specified on selected pages.
+      (chunk_name, data) = element
+      if data['datadump'] == []:
+         logging.info('USERLOG: chunk %s skipped.' % chunk_name)
```

Comment:
don't use % in logging statement:
 logging.info('USERLOG: chunk %s skipped.', chunk_name)